api/monitor.py

```diff
@@ -228,3 +234,49 @@ def get_identifiers(self, url=None):
         parsed_feed = feedparser.parse(feed)
         next_links = self.importer.extract_next_links(parsed_feed)
         return mapped_identifiers, next_links
+
+class LoanlikeReaperMonitor(ReaperMonitor):
+
+    @property
+    def where_clause(self):
+        """Find loans/holds that have either expired, or that were created a
+        long time ago and have no definite end date.
+        """
+        cls = self.MODEL_CLASS
+        end_field = getattr(cls, 'end')
+        start_field = getattr(cls, 'start')
+
+        now = datetime.datetime.utcnow()
+        expired = end_field < now
```

Comment:
I think for a Hold the end date is an estimate until it's available to check out, so an end date slightly in the past doesn't necessarily mean it's expired.